RealEstate/2 data.sql

```diff
@@ -0,0 +1,27 @@
+use RealEstate
+GO 
+truncate table Property
+go
+insert Property(
+    AddressNo,
+    Street,
+    Town,
+    PropertyType,
+    Bedrooms,
+    Bathrooms,
+    HouseSqFt,
+    LotSqFt,
+    [Owner],
```

Comment:
why is owner in brackets? Maybe we should name the column seller?

Reply:
Owner is a keyword in SQL Server. The brackets signify that it's an object name. You can put brackets around any object name like, `select [AddressNo] from `[RealEstate].[dbo].[Property]` and it's nothing bad and sometimes necessary. If you'd prefer `Seller` for other reasons then I can change!